controllers/members.js

```diff
@@ -139,11 +139,32 @@ const archiveMembers = async (req, res) => {
   }
 };
 
+/**
+ * Returns the lists of usernames where default archived role was added
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const addDefaultArchivedRole = async (req, res) => {
+  try {
+    const addedDefaultArchivedRoleData = await members.addDefaultArchivedRole();
```

Comment:
Should we be worried about potential timeout here. What happens if we have a million members?
Does this apply to members or to users?

Reply:
This applies to all users

Reply:
The approach I have in mind is to convert this into an async. api endpoint
First time we call this we will return an identifier for the async operation started on the server.
We can create a new collection and for each async operation we can create a document in it and store the progress of the operation in the doc 
Next time we can pass the identifier to check the status of the task

Reply:
By default firestore returns all documents in a collection if no limit is explicitly specifies

Docs link: https://firebase.google.com/docs/firestore/query-data/order-limit-data

Reply:
Currently there are a total of 656 users. I have tested with my personal firestore with 1000 users
It's taking around 5-10 seconds to [fetch all users + update all users archived role]
By default express has a timeout of 2 minutes. So timeout won't be a problem here